src/pages/certification/components/AuditorRunTestForm/AuditorRunTestForm.tsx

```diff
@@ -306,61 +316,38 @@ const AuditorRunTestForm: React.FC<IAuditorRunTestForm> = ({
         <div className={disable || verifying ? "disabled" : ""}>
           <div className="relative input-wrapper">
             <Input
-              label="GitHub Repository"
-              type="text"
-              id="repoURL"
-              required={true}
+              noGutter={true}
+              field={RepoField}
+              formState={form.formState}
+              register={form.register}
+              getFieldState={form.getFieldState}
               disabled={submitting || verifying}
-              tooltipText="Github Repository URL entered should be in the format - https://github.com/<username>/<repository> (with an optional trailing backslash)."
```

Comment:
This is missing in the revised component. We need this to let the user know what to enter in there

Reply:
Ok! I added it as a input placeholder